roaring/roaring.go

```diff
@@ -706,6 +706,27 @@ func (b *Bitmap) Check() error {
 	return a
 }
 
+func (b *Bitmap) Flip(start, end uint64) *Bitmap {
```

Comment:
I went down that route, to begin with, then I discovered the array flip function, given the range constraint, would be exactly the same as the higher level function and then I would get got the bitmap for "free" if just implemented at a higher level.  I suspect bitmap flip can be a bit more performant but I am skeptical of the magnitude of improvement.